### PR TITLE
feat:Adding placement options to Shellbar's menus

### PR DESCRIPTION
**Pull request by leo-ls**
## Description

implements #919

### CODE REVIEW

File: src/Shellbar/Shellbar.js [hunk 1]
```diff
@@ -469,7 +471,10 @@ Shellbar.propTypes = {
     logo: PropTypes.object,
     logoSAP: PropTypes.bool,
     notifications: PropTypes.object,
-    productMenu: PropTypes.array,
+    productMenu: PropTypes.shape({
+        items: PropTypes.array,
+        placement: PropTypes.oneOf(POPPER_PLACEMENTS)
```
Comment (skvale) [inline src/Shellbar/Shellbar.js:476]: As an alternative idea to having these be an object with `{items, placement}`, what about adding new props, like this:
```js
productMenuPlacement: PropTypes.oneOf(POPPER_PLACEMENTS),
profileMenuPlacement: PropTypes.oneOf(POPPER_PLACEMENTS)
```
Then it's not a breaking change and it seems more straightforward

File: src/Shellbar/Shellbar.js [hunk 2]
```diff
@@ -404,7 +406,7 @@ class Shellbar extends Component {
                                     }
                                     disableStyles={disableStyles}
                                     noArrow
-                                    placement='right'
+                                    placement={profileMenu.placement || 'right'}
```
Comment (skvale) [inline src/Shellbar/Shellbar.js:409]: Instead of doing `|| 'right'` let's add `'right'` as the defaultProp for this

File: .stylelintrc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "extends": ["stylelint-prettier/recommended"],
     "rules": {
-        "prettier/prettier": [true, {"singleQuote": true, "tabWidth": 4}]
+        "prettier/prettier": [true, {"singleQuote": true, "tabWidth": 4, endOfLine: "auto"}]
```
Comment (jbadan) [inline .stylelintrc:4]: Please remove the changes to this file

File: src/Shellbar/Shellbar.Component.js [hunk 4]
```diff
@@ -69,17 +69,21 @@ export class ShellbarComponent extends Component {
         userName: 'John Snow'
     };
 
-    profileMenu = [
-        { name: 'Settings', glyph: 'action-settings', size: 's', callback: () => alert('Settings selected!') },
-        { name: 'Sign Out', glyph: 'log', size: 's', callback: () => alert('Sign Out selected!') }
-    ];
+    profileMenu = {
```
Comment (jbadan) [inline src/Shellbar/Shellbar.Component.js:72]: Can you add examples to our storybook page as well?

**Comment by jbadan** [comment]
@leo-ls I'm going to close this pull request for lack of activity. Feel free to reopen once the changes requested are addressed 😃

**Pull request closed without merging**